SofaKernel/modules/SofaBaseMechanics/SubsetMapping.inl

```diff
@@ -169,9 +170,12 @@ void SubsetMapping<TIn, TOut>::init()
 
     topology = this->getContext()->getMeshTopology();
 
-    // Initialize functions and parameters
-    f_indices.createTopologicalEngine(topology);
-    f_indices.registerTopologicalData();
+    if (f_handleTopologyChange.getValue())
+    {
+        // Initialize functions and parameters for topological changes
+        f_indices.createTopologicalEngine(topology);
+        f_indices.registerTopologicalData();
+    }
```

Comment:
What happen if the Data<bool> f_resizeToModel is set to false, and the topology is still handling topologyChange? Will the list of indices of this subset still relevant ? Maybe you should add a condition on f_resizeToModel here.